build.ubuntu.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash 
+
+CPPREST_BUILD_DIR=build.$AC_PLATFORM
+if [[ -d ${CPPREST_BUILD_DIR} ]]; then  rm -rf CPPREST_BUILD_DIR ; fi
+mkdir ${CPPREST_BUILD_DIR}
+cd ${CPPREST_BUILD_DIR}
+pwd
+
+BOOST_ROOT=$HOME/projects/ac-access/.build_ubuntu/boost-1.61-ubuntu
```

Comment:
Please use some standard path (e.g. $HOME/boost-1.61-ubuntu or <cpprest_root>/boost-1.61-ubuntu). ac-access is not related to this project.

Reply:
Updated a patch here. https://github.com/anyconnect/cpprestsdk/pull/1/commits/97aa9e784f704d4bea5b1e5116026974e8ee3227